csharp/src/Ice/RemoteException.cs

```diff
@@ -33,6 +33,14 @@ namespace ZeroC.Ice
         /// <inheritdoc/>
         public override int GetHashCode() => HashCode.Combine(Retryable, Delay);
 
+        /// <inheritdoc/>
+        public override string? ToString() => Retryable switch {
+            Retryable.AfterDelay => $"after delay {Delay}",
```

Comment:
I think this should be $"after {Delay} delay", as in "after 50s delay".
The other switch cases don't need a "$".

Reply:
maybe use the `ToPropertyString` extension method too

`$"after {Delay.ToPropertyString()} delay"`

Reply:
Fixed